test-data/run.test

```diff
@@ -833,3 +833,30 @@ assert c() == 5.0
 assert d == 5.0
 assert e() == 5.0
 assert f == 5.0
+
+[case testArbitraryLvalues]
+from typing import List, Dict
+
+class O(object):
+    def __init__(self) -> None:
+        self.x = 1
+
```

Comment:
Also add a test that increment an attribute in an object that was defined in code we didn't compile works.
(That is, define a class in another file and pass in an object of it.)